queries/elvish/highlights.scm

```diff
@@ -0,0 +1,79 @@
+(comment) @comment
+
+(if "if" @conditional)
+(if (elif "elif" @conditional))
+(if (else "else" @conditional))
```

Comment:
Why not just `["if" "elif" "else"]`?

Reply:
Also for all the other keiwords

Reply:
You're right, I misunderstood how anonymous nodes work. I'm going to update this, but `else` should be in nested queries, because `if`/`while`/`for`/`try` can all have an `else` branch.